testsuite/devserver.go

```diff
@@ -363,7 +363,12 @@ func retryFor(maxAttempts int, interval time.Duration, cond func() error) error
 		} else {
 			lastErr = curE
 		}
-		time.Sleep(interval)
+		select {
+		case <-ctx.Done():
+			return ctx.Err()
+		case <-time.After(interval):
```

Comment:
Would prefer a ticker

Reply:
Updated

Reply:
:+1: Not a problem (ticker adjusts itself to slow receivers)